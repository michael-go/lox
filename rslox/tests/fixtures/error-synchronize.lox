print "before";
1+;
print "after";
foo(a, b, c