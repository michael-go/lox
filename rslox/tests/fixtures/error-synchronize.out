# exit code: 65
# stdout:

# stderr:
[line 2] Error at ';': Expect expression.
[line 4] Error at end: Expect ')' after arguments.

